diff --git a/node_modules/@remix-run/dev/compiler.js b/node_modules/@remix-run/dev/compiler.js
index eaa9bce..70c97c2 100644
--- a/node_modules/@remix-run/dev/compiler.js
+++ b/node_modules/@remix-run/dev/compiler.js
@@ -248,8 +248,8 @@ async function createServerBuild(config, options) {
       resolveDir: config.serverBuildDirectory
     },
     outfile: path.resolve(config.serverBuildDirectory, "index.js"),
-    platform: "node",
-    format: "cjs",
+    platform: process.env.SERVER_PLATFORM || "node",
+    format: process.env.SERVER_FORMAT || "cjs",
     target: options.target,
     inject: [reactShim],
     loader: loaders.loaders,
